sdk/src/derivation_path.rs

```diff
@@ -99,65 +108,158 @@ impl DerivationPath {
     }
 }
 
+impl fmt::Debug for DerivationPath {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "m")?;
+        for index in self.0.path() {
+            write!(f, "/{}", index)?;
+        }
+        Ok(())
+    }
+}
+
+fn bip44_solana_indexes() -> Vec<ChildIndex> {
+    vec![
+        ChildIndex::Hardened(BIP_44_PURPOSE),
+        ChildIndex::Hardened(BIP_44_COIN_SOL),
+    ]
+}
+
+fn child_index_from_str(s: &str) -> Result<ChildIndex, DerivationPathError> {
```

Comment:
If you like the above suggestions, I don't think this has any more callers

Reply:
Indeed! \o/